flathub.json

```diff
@@ -1,4 +0,0 @@
-{
-  "only-arches": ["x86_64"],
-  "disable-external-data-checker": true
```

Comment:
Unless maintainance resumes I suggest keeping the `disable-external-data-checker`. I had to disable it because PRs from the bot were piling up and using CI with no one to review or merge them.

Reply:
Yes, I will maintain it. It is now much simpler, so it should be not much work. I want to add Qt Designer Studio too, which is based on this changes.

Reply:
Ok fine then